### PR TITLE
Added Basil Kadha

### PR DESCRIPTION
**Pull request by somesh-ops**
Added Basil Kadha recipe.

### CODE REVIEW

File: src/recipes/basil-kadha.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            "ingredient": "Jaggery"
+        }
+    ],
+    "directions": [
+        "Wash the Basil, Lemongrass and Ginger.",
+        "Boil water in deep pan.",
+        "Add Basil, Lemongrass and Ginger and boil for 3-4 minutes",
+        "If the Panna is not very tangy, you can add some lemon juice to it.",
+        "Add jaggery and mix well then turn off the gas",
+        "Strain and serve hot.",
```
Comment (alfg) [inline src/recipes/basil-kadha.json:38]:
```suggestion
        "Strain and serve hot."
```

**Review by alfg — CHANGES_REQUESTED** [review]
Image seems to be missing from this PR. Also added a suggested fix for the invalid json.

**Review by alfg — CHANGES_REQUESTED** [review]
Please add the image. Thanks.

**Comment by somesh-ops** [comment]
> 
> 
> Please add the image. Thanks.

@alfg : Added, please review.

**Comment by alfg** [comment]
@somesh-ops can you remove the image from the old/invalid location?

**Comment by somesh-ops** [comment]
> 
> 
> @somesh-ops can you remove the image from the old/invalid location?

@alfg removed invalid upload, please review.

**Comment by alfg** [comment]
Thanks!

**Pull request merged**